package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "swsui",
-  "version": "0.1.0",
+  "version": "0.2.1",
```

Comment:
already 0.2.1? Because you ran some test-purpose releases?

Reply:
I had to get the proper files published to npm for mazz so it required several versions.
And also `@swift-sunshine/swsui` didn't work (paid feature only now) so I had to rollback to just `swsui`